provider/jira/jira.go

```diff
@@ -51,14 +51,20 @@ func (p *jiraSimple) Send(_ context.Context, notifier *domain.Notifier, body []b
 		return nil, err
 	}
 
+	reporter := (*Reporter)(nil)
```

Comment:
```suggestion
	reporter := new(Reporter)
```

Reply:
Tried this and it doesn't work as intended. `new(Reporter)` creates an empty `Reporter` struct and returns a pointer to it. Due to this when we convert the struct to JSON, since it's not `nil` it's included in the JSON. If we initialise `reporter` with `(*Reporter)(nil)`, it's value is `nil`. So when we convert the struct to JSON, since `reporter` is `nil` it is omitted from the JSON.